assets/src/edit-story/components/PrefixInput.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * External dependencies
+ */
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+
+const Label = styled.span`
+	color: ${ ( { theme } ) => theme.colors.fg.v5 };
+	font-size: 15px;
+  line-height: 16px;
+  text-align: center;
+	width: 35px;
+`;
+
+const Input = styled.input`
+  color: ${ ( { theme } ) => theme.colors.fg.v6 };
+  border: 0 !important;
+	font-size: 15px;
+	line-height: 16px;
+  width: 63px;
+  padding: 0 !important;
+  margin: 0;
+  
+  :focus {
+    outline: none !important;
+    box-shadow: none !important;
+  }
+`;
+
+const Group = styled.label`
+	color: ${ ( { theme } ) => theme.colors.mg.v1 };
+  display: flex;
+  align-items: center;
+  border: 1px solid ${ ( { theme } ) => theme.colors.fg.v3 };
+  border-radius: 4px;
+	width: 100px;
```

Comment:
Make it so that it will take up all the space available using `flex`. Sometimes this input will be alone on a line. This of course also then in turn applies to the input inside the group.

Reply:
flex-grow will just solve this issue. But in this case, for pixel perfect implementation, I think it could cause unexpected issues? On responsive side? For this case I think we can add new props to PrefixInput with flexGrow value.